mobilecoind/src/payments.rs

```diff
@@ -1181,6 +1365,15 @@ impl<T: BlockchainConnection + UserTxConnection + 'static, FPR: FogPubkeyResolve
             tx,
             outlay_index_to_tx_out_index,
             outlay_confirmation_numbers,
+            scis: scis
+                .iter()
+                .cloned()
+                .map(|(mut sci_for_tx, _sci_amount)| {
+                    // clear out proofs here since they were not part of the request
+                    sci_for_tx.sci.tx_in.proofs = Vec::default();
```

Comment:
I'm not sure this is necessary, maybe its better to include the proofs that were attached to each SCI? The end user can always clear them if they want to and we can document that the proofs are included in the response.

Reply:
I think they can also find the proofs in the `Tx` itself, right? they could find the TxIn that matches this SCI and then the proofs will be there.

It will make the response considerably smaller if we can avoid putting the proofs twice, and it makes it match what the user actually submitted -- that was why i went back and did it this way.

Reply:
Cool, sounds good